aws/aws_volumes.go

```diff
@@ -98,16 +95,14 @@ func (v *EbsVolumes) CreateSet(licensePath string, sizeOfEachVolume int, cluster
 	v.ClusterSize = fmt.Sprintf("%d", clusterSize)
 	v.SizeOfEachVolume = fmt.Sprintf("%d", sizeOfEachVolume)
 	v.LicensePath = licensePath
-	normalizedIOPS := v.iopsRatio * sizeOfEachVolume
-	if normalizedIOPS > 0 {
-		if normalizedIOPS < 100 {
-			normalizedIOPS = 100
-		}
-		if normalizedIOPS > 20000 {
-			normalizedIOPS = 20000
-		}
-	}
-	v.IoPs = fmt.Sprintf("%d", normalizedIOPS)
+
+	envVolType := os.Getenv("TF_VAR_stardog_home_volume_type")
+	if envVolType != "" && envVolType != "io1" {
+		v.IoPs = fmt.Sprintf("%d", 0)
+	} else {
+		v.IoPs = fmt.Sprintf("%d", sizeOfEachVolume * 50)
+	}
```

Comment:
We have *50 twice now.  Should that be a global variable?

Reply:
I guess it could be. Only will change if AWS changes that ratio.